CAN_Logger_with_Autobaud_and_Requests_no_CAN2_AES_CBC_SHA256/CAN_Logger_with_Autobaud_and_Requests_no_CAN2_AES_CBC_SHA256.ino

```diff
@@ -392,8 +411,8 @@ void check_buffer(){
     uint32_t start_micros = micros();
     
     // Write the beginning of each line in the 512 byte block
-    sprintf(prefix,"CAN3");
-    memcpy(&data_buffer[0], &prefix, 4);
+    sprintf(prefix,"CAN2");
```

Comment:
Dr. Daily I was just curious on the Prefix why did we change it from CAN3 to CAN2?

Reply:
The logfile format is CAN2. This is confusing, since we are using the same name for the hardware. The first logger didn't have multiple channels, so we had to add a byte in the data field to track the channel.